app/dashboard/templates/profiles/trust-vue.html

```diff
@@ -55,6 +55,9 @@ <h5>Active Now</h5>
                 Verify yourself on BrightID by proving you're human to other humans. Connect
                 with other users and join groups to meet the threshold.
                 <a href="https://brightid.org">Learn more.</a>
+                <br>
+                <br>
+                <a href={{ get_brightid_status }} class="text-nowrap">Refresh Status</a>
```

Comment:
add `"` to `href`

Reply:
the quotes escapes the html, without them the link renders correctly. [
<img width="386" alt="Screen Shot 2021-02-12 at 12 16 47 AM" src="https://user-images.githubusercontent.com/7246942/107732958-b7085380-6cc7-11eb-9b46-a76e18e9182e.png">
](url)